src/overlays/actors/ovl_En_Tana/z_en_tana.c

```diff
@@ -55,6 +55,8 @@ static void* sStoneTextures[] = {
 void EnTana_Init(Actor* thisx, GlobalContext* globalCtx) {
     EnTana* this = THIS;
 
+    //! @bug In the zora shop params is set to 2. This causes an out of bounds read on the array of shelf types. With
+    //! IDO it just happens to not crash, however with different compilers re-ordering may cause a crash.
```

Comment:
imo the bug tag makes more sense above the array itself, since the problem is really that its missing a string. the access is fine

Reply:
because of that I also think the details about zora shop being params 2 isnt necessarily important for the bug description. just the fact that its missing a third entry